Resources/doc/cookbook/recipe_knp_menu.rst

```diff
@@ -149,6 +149,38 @@ You can modify the menu via events easily. You can register as many listeners as
 
 Please see the `Using events to allow a menu to be extended`_ for further information.
 
+
+Hide menu item
+------------------
+
+You can modify the menu to hide some menu item. You need to add option 'show_in_dashboard' in your admin services or simple remove menu item from sonata_admin dashboard group configuration:
```

Comment:
> I don't find any lines with linebreak in this document. This document contains a line much more longer.

I know, they are all over the place :(

[I try to fix that](https://github.com/sonata-project/SonataAdminBundle/commit/d06f9568fca973e707f69a56687b46ebc9ae0590), but it is a huge job, so let's just not add new ones if we can, ok ?

> Is it some new rule about line length?

Yes. Try to keep them under 120 chars if you can, or even under 80 if possible.